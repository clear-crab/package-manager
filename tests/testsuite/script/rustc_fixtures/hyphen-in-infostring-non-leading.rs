--- Cargo-toml
---

// infostrings can contain hyphens as long as a hyphen isn't the first character.
//@ check-pass
// CARGO(fail): unsupported infostring

#![feature(frontmatter)]

fn main() {}
