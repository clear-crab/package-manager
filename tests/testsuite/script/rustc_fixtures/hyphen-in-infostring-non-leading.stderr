[ERROR] frontmatter infostring `Cargo-toml` is unsupported by cargo; specify `cargo` for embedding a manifest
